pkg/kubectl/cmd/apply.go

```diff
@@ -182,45 +182,17 @@ func RunApply(f cmdutil.Factory, cmd *cobra.Command, out io.Writer, options *App
 			return cmdutil.AddSourceToErr(fmt.Sprintf("retrieving modified configuration from:\n%v\nfor:", info), info.Source, err)
 		}
 
-		if err := info.Get(); err != nil {
```

Comment:
We should not remove all of this. Otherwise, we can't use `kubectl apply` to create non-existing resource.